test/libyul/yulOptimizerTests/fullSuite/stack_compressor_msize.yul

```diff
@@ -49,7 +49,7 @@
 //         sstore(not(gcd(10, 15)), 1)
 //         sstore(0, 0)
 //         sstore(2, 1)
-//         extcodecopy(1, msize(), 1, 1)
+//         pop(foo_singlereturn_1_78_248(calldataload(0), calldataload(3)))
```

Comment:
bad change.

Reply:
Can you check why this is happening? Does the linking function not fall into the "tiny" category in the full inliner?